### ISSUE TITLE
handle_navigation_error Navigate to error page returned

### ISSUE THREAD

**Issue by gabbycastilan** [pauleve/docker-mtgo#91]
I tried updating to wine 5.6 and followed the steps. I managed to get to step 5 wherein I need to install MTGO. However a Navigate to error page is returned. Kindly see this screenshot.

![Screen Shot 2020-04-18 at 8 55 03 PM](https://user-images.githubusercontent.com/63911261/79638393-9199de80-81b7-11ea-9c35-df8e700ebfa1.png)

**Comment by pauleve** [comment]
This is due to dotnet46 which is missing (step 4).  You may try to do `rm -rf ~/.wine*` to remove all previous installed files, and start again at step 2.

**Comment by gabbycastilan** [comment]
great! It is not returning the error anymore. However I am stucked again in step 5. Tried reinstalling everything from scratch a few times thinking I might have missed something and saw some lines after running step 4. 

Im not sure if this is the suspect?

![Screen Shot 2020-04-19 at 10 35 50 AM](https://user-images.githubusercontent.com/63911261/79678033-a4e59200-8229-11ea-92ce-d03b478eabd7.png)

**Comment by gabbycastilan** [comment]
I tried that and installed from scratch and it worked I think because it is not returning the same error again in Step 5.

However, I dont see any progress past that. I tried again installing everything and saw this error/warning returned after somewhere in Step4. 

![Screen Shot 2020-04-19 at 11 03 51 AM](https://user-images.githubusercontent.com/63911261/79678413-eaa45980-822d-11ea-8859-919219f50ea4.png)

Prompt said .Net 4.4 to 4.6 was successfully installed even if **The file '' was not found error** was displayed numerous times somewhere in the installation..

**Comment by pauleve** [comment]
In the last screenshot everything looks fine, this is the dialog which downloads the game client. What happens then?

**Comment by gabbycastilan** [comment]
nothing happens.. i tried leaving it for hours but no progress was made.

**Comment by pauleve** [comment]
Ah there is crash actually showing in the last screenshot.. yes, try with wine 5.5 staging https://dl.winehq.org/wine-builds/macosx/pool/winehq-staging-5.5.pkg 
you can also do the following for helping debugging:
```
wineserver -kw # shut down all wine processes
wine mtgo.exe &>output.log
```
and join the output.log file.

**Comment by gabbycastilan** [comment]
sure will do. Will keep you posted

**Comment by pauleve** [comment]
if you reinstall from scratch, you can use `winetricks -q` instead of `winetricks` at step 4 to suppress warnings.

**Comment by gabbycastilan** [comment]
So here were the steps I did:

1. Start from scratch, updated Homebrew
2. In Terminal, I ran rm `-rf ~/.wine*`
3. Installed Wine 5.5
4.  Used `wineserver -kw # shut down all wine processes
wine mtgo.exe &>output.log`. Now MTGO prompts installation
5. Able to Login BUT wont completely load assets. Then gets dcd

> Ah there is crash actually showing in the last screenshot.. yes, try with wine 5.5 staging https://dl.winehq.org/wine-builds/macosx/pool/winehq-staging-5.5.pkg
> you can also do the following for helping debugging:
> 
> ```
> wineserver -kw # shut down all wine processes
> wine mtgo.exe &>output.log
> ```
> 
> and join the output.log file.

Sorry, where do i find the output.log file?

**Comment by pauleve** [comment]
Do you get a crash? The file should be in the same directory as mtgo.exe

**Comment by gabbycastilan** [comment]
Just crashed...

![Screen Shot 2020-04-19 at 6 54 18 PM](https://user-images.githubusercontent.com/63911261/79685967-35929100-826f-11ea-9d38-f5bf963b6a71.png)


here are the logs leading to crash towards the end


[
[output.log](https://github.com/pauleve/docker-mtgo/files/4498815/output.log)
](url)

**Comment by pauleve** [comment]
not sure it has anything to do with wine.. this looks like an internet connectivity issue.. you may try with virtualbox or parallel to emulate a windows and give a try there ; it will likely be slow, but if you do not encounter this connectivity issue, then it will indicate a potential bug for wine.
Was it the same error with wine 5.6?

**Comment by gabbycastilan** [comment]
> not sure it has anything to do with wine.. this looks like an internet connectivity issue.. you may try with virtualbox or parallel to emulate a windows and give a try there ; it will likely be slow, but if you do not encounter this connectivity issue, then it will indicate a potential bug for wine.
> Was it the same error with wine 5.6?

ill try for 5.6... I have stable connection for sure since disconnection wasnt happening at all before this issue... but will try on virtual box or parallel. 

will keep you posted

**Comment by gabbycastilan** [comment]
@pauleve 

tried using wine v5.6 and its not even prompting to install MTGO. Attached is the log file using same version.


[Output log using Wine v5.6.txt](https://github.com/pauleve/docker-mtgo/files/4499688/Output.log.using.Wine.v5.6.txt)

**Comment by gabbycastilan** [comment]
**UPDATE:**

I tried both Wine v5.5 and v5.6, and also Parallels.  All installations were from scratch with cache for everything deleted first. Internet connection is stable, no hiccups.. but i am somehow UNABLE to connect with the server. Here are the observations:

**Wine v5.5** -- able to install MTGO, but unable to connect. 
**Wine v5.6** -- unable to install MTGO. Stuck at step5
**Parallels** -- able to install MTGO, but unable to connect.

In the screenshot, it is noted on MTGO startup that I am disconnected. This is common between v5.5 and Parallels. I was able to use this method in earlier version but I suddenly cant connect that's why I upgraded.

Hoping I could get this resolved :)

![parallel](https://user-images.githubusercontent.com/63911261/79765130-d01acf00-8358-11ea-9c91-7f6ab3e7ed94.png)

![wine v5 5](https://user-images.githubusercontent.com/63911261/79765343-13753d80-8359-11ea-88b5-3b211389fe0e.png)

[Wine v5.6 when wine mtgo.exe &>output.log is ran.txt](https://github.com/pauleve/docker-mtgo/files/4504199/Wine.v5.6.when.wine.mtgo.exe.output.log.is.ran.txt)
[Wine v5.5, disconnected on load of MTGO.txt](https://github.com/pauleve/docker-mtgo/files/4504197/Wine.v5.5.disconnected.on.load.of.MTGO.txt)
